lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -281,6 +284,20 @@ export abstract class ClientApplication {
         return this.tokenCache;
     }
 
+    /**
+     * @param state
```

Comment:
Please add a description here for typedoc purposes, including how this API can be used.